include/fmt/ranges.h

```diff
@@ -162,6 +162,33 @@ void for_each(Tuple &&tup, F &&f) {
   const auto indexes = get_indexes(tup);
   for_each(indexes, std::forward<Tuple>(tup), std::forward<F>(f));
 }
+
+template<typename Arg>
+FMT_CONSTEXPR const char* format_str_quoted(bool add_space, const Arg&, 
+    typename std::enable_if<!is_like_std_string<typename  std::decay<Arg>::type>::value>::type* = nullptr) {
```

Comment:
nit: could you wrap long (>80 char length) lines per the style guide (https://google.github.io/styleguide/cppguide.html#Line_Length) here and below?